2-engineering-principles.md

```diff
@@ -91,7 +91,7 @@ part of our safety net. We optimize for
 We deliberately minimize the variety of technologies in our system to drive
 efficiency, and, at the same time, have a managed program of innovation. To
 achieve this, we monitor actual technologies in use, gravitate to those most
-appropriately addressing the given challenges and selectively experimenting with
+appropriately addressing the given challenges and selectively experiment with
```

Comment:
Hm "experimenting" does not sound to wrong for me. @lorenzoernesto to the rescue please

Reply:
I think it has to be read like this:
> To achieve this, we [...] selectively experiment with [...]

Reply:
To be type safe (and correct English), the form of "monitor" and "gravitate" has to agree with the form of "experiment". It might look like "experimenting" needs to agree with "addressing", but it doesn't.

To make it super clear, I would suggest (changes in bold):

> To achieve this, we monitor **current** technologies in use, gravitate to those **that** most appropriately **address** the given challenges and selectively **experiment** with new technologies to find those that will improve our products.